src/base/runner.ts

```diff
@@ -12,18 +12,7 @@ const Runners: RunnersType = {
   redis: RedisRunner,
 };
 
-const getRunner = ({
-  config,
-  registry,
-  pool,
-  logger,
-  hooks,
-}: {
-  config: Configuration;
-  registry: IRegistry;
-  pool: Pool<any>;
-  logger: Logger;
-  hooks?: Hooks;
-}) => new Runners[config.engine]({ config, registry, pool, logger, hooks });
+const getRunner = ({ steveo }: { steveo: Steveo }) =>
```

Comment:
Should these just be `steveo: Steveo`? think we've added an object wrapper in what I assume was just meant to be fixing the types (same thing in all the runner constructors)